spec/lib/pre_assembly/bundle_spec.rb

```diff
@@ -158,14 +167,15 @@
       end
     end
 
-    it "exclude_from_content() should behave correctly" do
-      expect(rumsey.exclude_from_content(rumsey.path_in_bundle('image1.tif'))).to be_falsey
-      expect(rumsey.exclude_from_content(rumsey.path_in_bundle('descMetadata.xml'))).to be_truthy
-    end
+    # it "exclude_from_content() should behave correctly" do
+    #   expect(rumsey.exclude_from_content(rumsey.path_in_bundle('image1.tif'))).to be_falsey
+    #   expect(rumsey.exclude_from_content(rumsey.path_in_bundle('descMetadata.xml'))).to be_truthy
+    # end
```

Comment:
file a ticket about re-doing this test?  Is the basic functionality being tested here still in scope?

Reply:
my recollection of an overheard conversation yesterday is that it can go altogether for the webapp (though if it's projected that we'll need it in the future, and we already have the logic in `Bundle` for except/only behavior, i'd be inclined to leave it in, and at least ticket re-enabling these tests).  i'd also mark `skip` instead of commenting out, if we wanted to not remove it, but didn't have time to fix it.

Reply:
#227